include/kernels/staggered_coarse_op_kernel.cuh

```diff
@@ -1,14 +1,21 @@
-#include <color_spinor_field_order.h>
 #include <gauge_field_order.h>
 #include <multigrid_helper.cuh>
-#include <index_helper.cuh>
-#include <gamma.cuh>
+#include <kernel.h>
 
 namespace quda {
 
-  template <typename Float, int coarseSpin, int fineColor, int coarseColor,
-            typename coarseGauge, typename fineGauge>
-  struct CalculateStaggeredYArg {
+  template <typename Float_, int coarseSpin_, int fineColor_, int coarseColor_,
+            typename coarseGauge, typename fineGauge, bool kd_build_x_ = false>
+  struct CalculateStaggeredYArg : kernel_param<1> {
```

Comment:
`kernel_param<1>` -> `kernel_param<>`, 1 is the default

Reply:
276e11f06c3b58c4a88398c334414ff5b8c9a907